pkg/segment/results/mresults/seriesresult.go

```diff
@@ -777,6 +777,16 @@ func reduceEntries(entries []Entry, fn utils.AggregateFunctions, fnConstant floa
 		} else {
 			ret = entriesCopy[int(index)].dpVal
 		}
+	case utils.Stddev:
+		fallthrough
+	case utils.Stdvar:
+		sum := 0.0
+		for i := range entries {
+			sum += entries[i].dpVal
+		}
+		ret = sum / float64(len(entries))
+	case utils.Group:
```

Comment:
How is the `utils.Stddev` case correct? Don't you have to take the square root?

Reply:
This reduce logic is the downsample logic not the aggregation logic. We downsample data points to each timestamp. 
For example: 

```
1. seriesId: {color: red}, timestampToVals: {1: 3, 2: 4, 5: 5...}
2. seriesId: {color: yellow}, timestampToVals: {1: 4, 2: 5, 5: 6...}

Downsample interval is 1 h.
So After Downsample, they will be:
1. seriesId: {color: red}, timestampToVals: {0: 4}
2. seriesId: {color: yellow}, timestampToVals: {0: 5}

Then do aggregation ...

```